pkg/reconciliation/reconcile_racks.go

```diff
@@ -1616,12 +1650,32 @@ func (rc *ReconciliationContext) ReconcilePods(statefulSet *appsv1.StatefulSet)
 			rc.Recorder.Eventf(rc.Datacenter, corev1.EventTypeNormal, events.LabeledRackResource,
 				"Update rack labels for PersistentVolumeClaim %s", pvc.Name)
 		}
+		pvcAnns := pvc.GetAnnotations()
+		oplabels.AddOperatorAnnotations(pvcAnns, rc.Datacenter)
+		if !reflect.DeepEqual(pvcAnns, pvc.GetAnnotations()) {
+			rc.ReqLogger.Info("Updating annotations",
+				"PVC", pvc,
+				"current", pvc.GetAnnotations(),
+				"desired", pvcAnns)
+			pvc.SetAnnotations(pvcAnns)
+			pvcPatch := client.MergeFrom(pvc.DeepCopy())
+			if err := rc.Client.Patch(rc.Ctx, pvc, pvcPatch); err != nil {
+				rc.ReqLogger.Error(
+					err,
+					"Unable to update pvc with annotation",
+					"PVC", pvc,
+				)
+			}
+
+			rc.Recorder.Eventf(rc.Datacenter, corev1.EventTypeNormal, events.LabeledRackResource,
+				"Update rack annotations for pvc %s", pvc.Name)
+		}
 	}
 
 	return nil
 }
 
-func mergeInLabelsIfDifferent(existingLabels, newLabels map[string]string) (bool, map[string]string) {
+func mergeInTagsIfDifferent(existingLabels, newLabels map[string]string) (bool, map[string]string) {
```

Comment:
What are tags? Why is this renamed to something weird?

Reply:
The 'tags' word comes from k8ssandra-operator, I wanted to re-use the same function for both annotations and labels, which means we need some other word that covers both.

So 
1. Is this a nit, issue, suggestion, or..?
2. Feel free to suggest a better word if you can think of one, and if you think it is worth moving away from the k8ssandra-operator way of naming things.

Reply:
This isn't used for annotations, not anywhere in your PR (the only change is modifying name of this function). This function has had its name for 4 years now, k8ssandra-operator hasn't even existed that long.

This change shouldn't be in this PR. Is it a leftover of something that should be here, or something that's just dead code change that wasn't necessary?

Reply:
Hmmm, I see your point having taken a closer look. Looks like (as you say) this is from a previous approach I tried rather than being strictly required. I will change it back.